main.go

```diff
@@ -287,19 +132,19 @@ func installVersion(arg string, custBinPath *string, mirrorURL *string) {
 		installFileVersionPath := lib.ConvertExecutableExt(filepath.Join(installLocation, versionPrefix+requestedVersion))
 		recentDownloadFile := lib.CheckFileExist(installFileVersionPath)
 		if recentDownloadFile {
-			lib.ChangeSymlink(installFileVersionPath, *custBinPath)
+			lib.ChangeSymlink(installFileVersionPath, customBinaryPath)
 			fmt.Printf("Switched terraform to version %q \n", requestedVersion)
 			lib.AddRecent(requestedVersion) //add to recent file for faster lookup
 			os.Exit(0)
 		}
 
 		//if the requested version had not been downloaded before
-		listAll := true                                     //set list all true - all versions including beta and rc will be displayed
-		tflist, _ := lib.GetTFList(*mirrorURL, listAll)     //get list of versions
+		//set list all true - all versions including beta and rc will be displayed
+		tflist, _ := lib.GetTFList(mirrorURL, true)         //get list of versions
 		exist := lib.VersionExist(requestedVersion, tflist) //check if version exist before downloading it
```

Comment:
While you're here please
```suggestion
		exist := lib.VersionExist(requestedVersion, tflist) // Check if version exists before downloading it
```